app/controllers/idv/session_errors_controller.rb

```diff
@@ -36,8 +36,6 @@ def ssn_failure
 
     def throttled
       @expires_at = Throttle.new(user: effective_user, throttle_type: :idv_doc_auth).expires_at
-      analytics.throttler_rate_limit_triggered(throttle_type: :idv_doc_auth)
```

Comment:
I noticed your comment mentioned these changes as being somewhere that was starting to log duplicates. Does that apply to every place where we redirect here? Since [it's referenced in quite a few places](https://github.com/18F/identity-idp/search?q=idv_session_errors_throttled_url).

Reply:
@eric-gade and I dug into this and the additional lines in SessionErrorsController#throttled had added double logging in those other places. Now everything is being logged once as it should be.

It turns out that that have_received expects exactly one event, where receive and have_logged accept one or more, so that's why tests didn't catch the duplication.